README.md

```diff
@@ -49,86 +51,99 @@ on, we suggest opening an issue detailing your ideas first.
 
 Otherwise send us a pull request and we will be happy to review it.
 
-### Code Style
-
-All code should be formatted through `go fmt`. This tool will automatically
-format code for you. Although it is recommended you write code in this style
-and just use this tool to catch mistakes.
+#### 1. Install Dependencies
 
-### Building
```

Comment:
I prefer the old wording here too mostly. Although sub categories for testing and stuff is a good idea.

Reply:
OK, no problem. Which parts do you think read better with the original wording?

Reply:
I made a branch on top of yours https://github.com/Morganamilo/yay/tree/%23661. Feel free to critique my critique. The commits are split up to explain my rational.